lib/core/src/Cardano/Wallet/Api/Lib.hs

```diff
@@ -0,0 +1,47 @@
+
+
+{-# LANGUAGE DerivingVia #-}
+
+module Cardano.Wallet.Api.Lib
+    ( eitherToParser
+    , hexText
+    , fromHexText
+    , toTextJSON
+    , fromTextJSON
+    )
+    where
+
+import Prelude
+
+import Cardano.Wallet.Primitive.AddressDerivation
+    ( fromHex, hex )
+import Cardano.Wallet.Util
+    ( ShowFmt (..) )
+import Data.Aeson
+    ( ToJSON (..), Value, withText )
+import Data.Bifunctor
+    ( Bifunctor (..) )
+import Data.ByteString
+    ( ByteString )
+import Data.Text
+    ( Text )
+import Data.Text.Class
+    ( FromText (..), ToText (toText) )
+
+import qualified Data.Aeson.Types as Aeson
+import qualified Data.Text.Encoding as T
+
+eitherToParser :: Show s => Either s a -> Aeson.Parser a
+eitherToParser = either (fail . show) pure
+
+hexText :: ByteString -> Text
+hexText = T.decodeLatin1 . hex
+
+fromHexText :: Text -> Either String ByteString
+fromHexText = fromHex . T.encodeUtf8
```

Comment:
I was thinking that these 2 functions go to the `Cardano.Wallet.Api.Hex` and other 3 functions go to the `Cardano.Wallet.Api.Json`. This way we avoid catch-all names like "Util", "Helpers", "Lib" etc.

Reply:
I will check if it's useful to take them out, probably it's not and I will take them back to Api.Types